src/any_agent/tracing/trace.py

```diff
@@ -182,6 +182,15 @@ def get_total_cost(self) -> TotalTokenUseAndCost:
             total_cost_completion=total_cost_completion,
         )
 
+    def add_span(self, span: AgentSpan) -> None:
```

Comment:
I guess do you plan to expand this somehow?

Reply:
This ties into the conversation I was having with Javier on slack: if we go down the route of keeping all the important information inside of OTEL, then we would maybe need to add an attribute for things like `agent_framework` into every span that we add. Still thinking about it.